alicloud/resource_alicloud_oss_bucket.go

```diff
@@ -204,6 +219,31 @@ func resourceAlicloudOssBucket() *schema.Resource {
 								},
 							},
 						},
+						"noncurrent_version_transition": {
```

Comment:
这个接口实现有问题。
一个rule 可以设置多个 noncurrent_version_transition 条件。

Reply:
Hello @huiguangjun , it seems like the go sdk currently does not support the above said configuration. I have made changes as per the go sdk. Here is the goDoc for Lifecycle Rules struct - https://godoc.org/github.com/aliyun/aliyun-oss-go-sdk/oss#LifecycleRule

The `LifecycleRule` struct currently takes in 1 object for `NonVersionTransition` .

Also, it would be great if you continue conversation in english :) ! I have to translate the above comment :P

Reply:
@huiguangjun  any updates on this ? Let me know your thoughts.